apps/trustlab/src/payload/access/abilities.js

```diff
@@ -0,0 +1,67 @@
+import { checkRole } from "./checkRole";
+import { ROLE_ADMIN, ROLE_AUTHOR, ROLE_DEFAULT, ROLE_EDITOR } from "./roles";
+
+export const canManageContent = (user) => {
+  if (!user) return false;
+  const isAdmin = checkRole([ROLE_ADMIN], user);
+  if (isAdmin) return true;
```

Comment:
```suggestion
  if (isAdmin) {
    return true;
  }
```

Write full `if`'s with `{ }`. Aren't `prettier`/`eslint` configured for this?